src/Mod/Sketcher/Gui/ViewProviderSketch.cpp

```diff
@@ -3082,7 +3084,11 @@ QString ViewProviderSketch::getPresentationString(const Constraint *constraint)
             }
         }
     }
-
+    if(constraint->Type == Sketcher::Diameter){
+        userStr.insert(0,QChar(8960));
+    }else if(constraint->Type == Sketcher::Radius){
+        userStr.insert(0,QChar(82));
+    }

```

Comment:
I know there are no hard code convention rules in FC, but the current consensus is this formatting:

```
if (constraint->Type == Sketcher::Diameter){
        userStr.insert(0, QChar(8960));
}
else if (constraint->Type == Sketcher::Radius){
        userStr.insert(0, QChar(82));
}
```

So a space behind "if" and "else",
Else begins on a new line
there is a space behind a comma.

Besides this, please add some comments what your code does. For example what is QChar(8960). You could add a comment that this is the char '⌀' (U+2300).

Reply:
done